types/polka/polka-tests.ts

```diff
@@ -1,16 +1,9 @@
-import * as compression from 'compression';
 import * as Polka from 'polka';
 
-interface MyResponse {
-    foo: string;
-}
-
-const middleware: Polka.Middleware<any, MyResponse, any, any> = async (req, res, next) => {
+const middleware: Polka.Middleware = async (req, res, next) => {
     const originalUrl = req.originalUrl;
     const path = req.path;
 
-    res.send({ foo: 'bar' });
```

Comment:
The method `res.send()` is coming from express types. It doesn't exist in polka, see https://github.com/lukeed/polka/blob/master/packages/polka/index.js